tsconfig.json

```diff
@@ -1,9 +1,3 @@
 {
-  "extends": "./tsconfig.base.json",
```

Comment:
Did you mean to change these tsconfig files?

Reply:
Fixed it, the new build system has a bug. Trying to sort it out.